### ISSUE TITLE
Tree tab wtf?

### ISSUE THREAD

**Issue by tj** [apex/apex-ui#24]
not sure where this came from, but it's bumping the tree down

<img width="348" alt="screen shot 2017-06-12 at 7 55 37 pm" src="https://user-images.githubusercontent.com/25254/27063881-25130efa-4fa9-11e7-9073-c8271694f799.png">

**Comment by tj** [comment]
hmm the active tab is no longer black either.

**Comment by tj** [comment]
seems like a state issue in Atom, it thinks the active pain is not active

**Comment by luca-aurelia** [comment]
Yeah, here's the relevant Atom issue for the active pane problem:
https://github.com/atom/atom/issues/14283

One work-around is to open and close a pane when Atom starts:

```CoffeeScript
openNewPane = ->
  atom.workspace.open(null, { split: "right" })
closeActivePane = ->
  atom.workspace.getActiveTextEditor().destroy()
defer = (fn) ->
  setTimeout fn, 1
openThenClose = ->
  openNewPane().then ->
    defer closeActivePane

defer openThenClose
```
